projects/packages/classic-theme-helper/src/responsive-videos.php

```diff
@@ -0,0 +1,175 @@
+<?php
+/**
+ * Theme Tools: Responsive videos enhancements.
+ *
+ * @package automattic/jetpack
+ */
+
+use Automattic\Jetpack\Assets;
+
+/**
+ * Load the Responsive videos plugin
+ */
+function jetpack_responsive_videos_init() {
+
+	/* If the doesn't theme support 'jetpack-responsive-videos', don't continue */
+	if ( ! current_theme_supports( 'jetpack-responsive-videos' ) ) {
+		return;
+	}
+
+	/* If the theme does support 'jetpack-responsive-videos', wrap the videos */
+	add_filter( 'wp_video_shortcode', 'jetpack_responsive_videos_embed_html' );
+	add_filter( 'video_embed_html', 'jetpack_responsive_videos_embed_html' );
+
+	/* Only wrap oEmbeds if video */
+	add_filter( 'embed_oembed_html', 'jetpack_responsive_videos_maybe_wrap_oembed', 10, 2 );
+	add_filter( 'embed_handler_html', 'jetpack_responsive_videos_maybe_wrap_oembed', 10, 2 );
+
+	/* Wrap videos in Buddypress */
+	add_filter( 'bp_embed_oembed_html', 'jetpack_responsive_videos_embed_html' );
+
+	/* Wrap Slideshare shortcodes */
+	add_filter( 'jetpack_slideshare_shortcode', 'jetpack_responsive_videos_embed_html' );
+
+	// Remove the Jetpack Responsive video wrapper in embed blocks on sites that support core Responsive embeds.
+	if ( current_theme_supports( 'responsive-embeds' ) ) {
+		add_filter( 'render_block', 'jetpack_responsive_videos_remove_wrap_oembed', 10, 2 );
+	}
+}
+add_action( 'after_setup_theme', 'jetpack_responsive_videos_init', 99 );
+
+/**
+ * Adds a wrapper to videos and enqueue script
+ *
+ * @param string $html The video embed HTML.
+ * @return string
+ */
+function jetpack_responsive_videos_embed_html( $html ) {
+	if ( empty( $html ) || ! is_string( $html ) ) {
+		return $html;
+	}
+
+	// Short-circuit for AMP responses, since custom scripts are not allowed in AMP and videos are naturally responsive.
+	if ( class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request() ) {
+		return $html;
+	}
+
+	// The customizer video widget wraps videos with a class of wp-video
+	// mejs as of 4.9 apparently resizes videos too which causes issues
+	// skip the video if it is wrapped in wp-video.
+	$video_widget_wrapper = 'class="wp-video"';
+
+	$mejs_wrapped = strpos( $html, $video_widget_wrapper );
+
+	// If this is a video widget wrapped by mejs, return the html.
+	if ( false !== $mejs_wrapped ) {
+		return $html;
+	}
+
+	Assets::register_script(
```

Comment:
This is the only place I needed to change things from the original file. Since the package gives us the opportunity to "start fresh" I didn't use all the legacy code to build the *.min.js version of the file and instead I used the standard (new) build script.